### PR TITLE
FunnierFacts.txt

### PR DESCRIPTION
**Pull request by Feel65**

### CODE REVIEW

File: docs/assets/funfacts.txt [hunk 1]
```diff
@@ -15,18 +15,18 @@ Grass is a plant with narrow leaves growing from the base. [...]\nGrasses are mo
 SkyClient was once exclusive to Windows users.\nFortunately, now we have a Java-based version, and a web-based version [WIP].
 You'll never believe this, but our Discord's support channel\nis called #support! Use it as necessary.
 SBE has skidded from countless mods including DSM, Skytils and NEU!
-SkyClient contains only the finest ethically sourced mods and packs -\nwe asked permission before adding them to SkyClient.
+SkyClient contains only the finest ethically sourced mods and packs -\nwe asked permission before adding them to SkyClient, so the child authors wouldn't get mad.
 SkyClient is for more than Skyblock, even though it started out that way.
-SkyClient is a zero profit organization; we have 40 unpaid volunteers around the globe.
-Most mod's settings are located in a command you can run when in game;\ne.g Patcher is configurable with `/patcher`.
+SkyClient is a non-profit organization; we have 40 unpaid volunteers around the globe.
+Most mods settings are located in a command you can run when in game;\ne.g Patcher is configurable with `/patcher`.
```
Comment (KTibow) [inline docs/assets/funfacts.txt:21]: "mods'"?

File: docs/assets/funfacts.txt [hunk 2]
```diff
@@ -15,18 +15,18 @@ Grass is a plant with narrow leaves growing from the base. [...]\nGrasses are mo
...
 At one point in Minecraft, there was a rare chance of Snow Golems naturally spawning.
 When you're asked to update your mods, you can click on a mod's name to stop it\nfrom being replaced with SkyClient's latest version.
-Wyvest
+Whyvest
```
Comment (KTibow) [inline docs/assets/funfacts.txt:24]: real

File: docs/assets/funfacts.txt [hunk 3]
```diff
@@ -15,18 +15,18 @@ Grass is a plant with narrow leaves growing from the base. [...]\nGrasses are mo
 SkyClient was once exclusive to Windows users.\nFortunately, now we have a Java-based version, and a web-based version [WIP].
 You'll never believe this, but our Discord's support channel\nis called #support! Use it as necessary.
 SBE has skidded from countless mods including DSM, Skytils and NEU!
-SkyClient contains only the finest ethically sourced mods and packs -\nwe asked permission before adding them to SkyClient.
+SkyClient contains only the finest ethically sourced mods and packs -\nwe asked permission before adding them to SkyClient, so the child authors wouldn't get mad.
 SkyClient is for more than Skyblock, even though it started out that way.
-SkyClient is a zero profit organization; we have 40 unpaid volunteers around the globe.
-Most mod's settings are located in a command you can run when in game;\ne.g Patcher is configurable with `/patcher`.
+SkyClient is a non-profit organization; we have 40 unpaid volunteers around the globe.
```
Comment (KTibow) [inline docs/assets/funfacts.txt:20]: we're not an official registered organization, i liked how it was before

File: docs/assets/funfacts.txt [hunk 4]
```diff
@@ -15,18 +15,18 @@ Grass is a plant with narrow leaves growing from the base. [...]\nGrasses are mo
 SkyClient was once exclusive to Windows users.\nFortunately, now we have a Java-based version, and a web-based version [WIP].
 You'll never believe this, but our Discord's support channel\nis called #support! Use it as necessary.
 SBE has skidded from countless mods including DSM, Skytils and NEU!
-SkyClient contains only the finest ethically sourced mods and packs -\nwe asked permission before adding them to SkyClient.
+SkyClient contains only the finest ethically sourced mods and packs -\nwe asked permission before adding them to SkyClient, so the child authors wouldn't get mad.
```
Comment (KTibow) [inline docs/assets/funfacts.txt:18]: what
no

**Review by KTibow — CHANGES_REQUESTED** [review]
aside from these, also check that they aren't too long, that's why i had to add the `\n`

**Comment by KTibow** [comment]
[testing webhook]

**Review by nacrt — APPROVED** [review]

File: docs/assets/funfacts.txt (hunk 1 above)
Comment (Feel65) [reply docs/assets/funfacts.txt]: Most “mod is” settings

Vs

Most mods settings


Grammar

File: docs/assets/funfacts.txt (hunk 2 above)
Comment (Feel65) [reply docs/assets/funfacts.txt]: whyvest

File: docs/assets/funfacts.txt (hunk 3 above)
Comment (Feel65) [reply docs/assets/funfacts.txt]: zero profit just feels wrong tho

File: docs/assets/funfacts.txt (hunk 4 above)
Comment (Feel65) [reply docs/assets/funfacts.txt]: shush you can change it

**Pull request merged**